src/main/java/gov/usgs/earthquake/nats/NATSStreamingNotificationSender.java

```diff
@@ -0,0 +1,86 @@
+package gov.usgs.earthquake.nats;
+
+import gov.usgs.earthquake.distribution.*;
+import gov.usgs.util.Config;
+
+import javax.security.auth.login.Configuration;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+/**
+ * Sends notifications directly to NATS streaming server using a NATS client
+ */
+public class NATSStreamingNotificationSender extends DefaultNotificationSender {
+
+  private static final Logger LOGGER = Logger
+          .getLogger(DefaultNotificationSender.class.getName());
+
+  private NATSClient client = new NATSClient();
+  private String subject;
+
+  @Override
+  public void configure(Config config) throws Exception{
+    super.configure(config);
+    client.configure(config);
+    subject = config.getProperty(NATSClient.SUBJECT_PROPERTY);
```

Comment:
What about a default value?

Reply:
The argument for the low-level NATS senders is that default values would be poorly targeted, since they'll probably be very different for every hub. We could use USGS defaults, but next to nobody would use them.

Reply:
For now, most hubs would have their own nats server.  Only when we share a nats cluster across many hubs will that need to be overridden...

Reply:
Added reasonable-ish defaults in node-ws branch.